tests/sles4sap/ensa/netweaver_ensa2_web_methods.pm

```diff
@@ -61,6 +73,11 @@ sub run {
     $self->webmethod_checks($instance_id);
     # Execute failover from ASCS instance - this will return resources to original host
     if ($instance_type eq 'ASCS') {
+        # Some delay needed here for 12-SP5, don't know why even webmethod_checks passed
```

Comment:
I guess nothing in `->webmethod_checks` nor in `->sapcontrol` kills the test. I wonder if it's needed. @lpalovsky WDYT?

Reply:
It is needed, see PR's description:
`The "sleep 300" before doing "failover" is the **key** fix. (I also tried sleep 60, sleep 120 but don't work. So chose sleep 300 and it works. I can not explain why)`

Reply:
It is needed, see PR's description:
`The "sleep 300" before doing "failover" is the **key** fix. (I also tried sleep 60, sleep 120 but don't work. So chose sleep 300 and it works. I can not explain why)`